source/common.c

```diff
@@ -18,6 +18,10 @@
 #include <stdarg.h>
 #include <stdlib.h>
 
+#ifdef _WIN32
+#    include <Windows.h>
+#endif
```

Comment:
the code that uses SecureZeroMemory branches on _MSC_VER

Reply:
Marco is right. _byteswap should be _MSC_VER, this should be _WIN32.